examples/README.md

```diff
@@ -16,7 +18,7 @@
 
 ```bash
 cd ebpf/examples/
-go run -exec sudo [./kprobe, ./uretprobe, ./ringbuffer, ...]
+go run -exec sudo [./kprobe, ./uretprobe, ./ringbuffer, ./cgroup_skb, ...]
```

Comment:
This extension is not needed, it's just to indicate how the examples can be run.